modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/adapter/SW360ComponentClientAdapter.java

```diff
@@ -52,9 +56,13 @@ public SW360Component createComponent(SW360Component component, HttpHeaders head
     }
 
     public Optional<SW360Component> getComponentByArtifact(Artifact artifact, HttpHeaders header) {
-        String componentName = SW360ComponentAdapterUtils.createComponentName(artifact);
-
-        return getComponentByName(componentName, header);
+        try {
+            String componentName = SW360ComponentAdapterUtils.createComponentName(artifact);
+            return getComponentByName(componentName, header);
+        } catch (ExecutionException e) {
+            LOGGER.debug("No component found for {}. Reason: {}", artifact, e.getMessage());
```

Comment:
Always log the exception, especially debug messages are meant for us to understand what is going on, if a user comes back to us, so don't be shy and print out whatever you think could help us in identifying the problem.